hazelcast/src/test/java/com/hazelcast/wan/impl/WanPublisherMigrationTest.java

```diff
@@ -137,8 +140,12 @@ public void testMigration() {
                     publisher.migrationProcess.intValue() >= partitionsToMigrate);
             assertEquals(exceptionMsg("migrationCommit", publisher),
                     partitionsToMigrate, publisher.migrationCommit.intValue());
```

Comment:
What about this assert - doesn't it suffer the same issue?

Reply:
That also needed to be change. I've missed it. Thanks!